rpcs3/Emu/RSX/rsx_methods.cpp

```diff
@@ -2769,6 +2769,23 @@ namespace rsx
 		methods[NV3089_IMAGE_IN_OFFSET]                   = nullptr;
 		methods[NV3089_IMAGE_IN]                          = nullptr;
 
+		// Unknown, used by Dragon Ball: Raging Blast 2 [BLES00978] and J-Stars Victory VS [BLJS10258]
+		methods[0xac40 >> 2]                              = nullptr;
```

Comment:
This is obviously one method, bind it with bind_array. I remember debugging this a long time ago, its something to do with textures, I just don't know what.

Reply:
I added some debugging info for invalid methods (start method and count).
I should wait until a new, updated log is recieved.
For now im removing this commit from the pr.

Reply:
Ok then. Although I do think its a bug in the game and someone initialized a wrong argument count. Or the game is developed using an old SDK with a bug in this region.